cf/commands/space/space_test.go

```diff
@@ -112,6 +112,16 @@ var _ = Describe("space command", func() {
 			})
 		})
 
+		Context("when the security-group-rules flag is passed", func() {
+			It("it shows space information and security group rules", func() {
+				runCommand("--security-group-rules", "whose-space-is-it-anyway")
+
+				Expect(ui.Outputs).To(ContainSubstrings(
+					[]string{"Getting rules for the security group"},
```

Comment:
Looks like the test only cover the title of the table. We would like the test to also assert the content of the table is correctly printed. Do you mind updating the test a little?